src/tests/utils.cairo

```diff
@@ -19,11 +19,16 @@ fn deploy(contract_class_hash: felt252, calldata: Array<felt252>) -> ContractAdd
 
 /// Pop the earliest unpopped logged event for the contract as the requested type
 /// and checks there's no more data left on the event, preventing unaccounted params.
-/// Indexed event members are currently not supported, so they are ignored.
+/// This function also removes the hashed event-name key to support indexed event
+/// members.
```

Comment:
i don't understand, what's being removed and why removing it supports indexed members?

Reply:
> why removing it supports indexed members?

Because the first key is always the event ID (`sn_keccak('EventName')`)

> might be worth explaining what the "hashed event-name key" is

I simplified it to `event ID`. IMO we don't need the specifics, but I'm happy to expand on the (hopefully fixed) comment if you think it'll be helpful

Reply:
it looks better now